packages/design/src/Colors.mdx

```diff
@@ -24,4 +24,135 @@ ensure you are using colors for their intended purpose.
   alongside color to ensure your content can be understood by everyone.
 </Banner>
 
+## Usage Guidelines
+
+### Typography
+
+Typographic elements in Jobber should use consistent colors to ensure
+readability.
+
+#### Heading
+
+Headings have a bold, dark color to cement their hierarchy.
+
+#### Text
+
+A slightly softer color is used for body text for greater readability.
+
+##### Text--Supporting
+
+Text that is relevant but less important can be lower-contrast to suggest its’
+reduced importance.
+
+### Interactive
+
+Use these colors in buttons and form controls to communicate the presence and
+meaning of interaction.
+
+#### Interactive
+
+The default color used for interactive elements.
```

Comment:
There is double `Interactive` headings. Is that on purpose?

Reply:
Yep, Interactive is the category and this interactive color is the default color for interactive elements